packages/dashboard-frontend/src/store/__mocks__/storeBuilder.ts

```diff
@@ -158,29 +144,9 @@ export class FakeStoreBuilder {
     isLoading = false,
     error?: string,
   ): FakeStoreBuilder {
-    this.state.dwDockerConfig.registries = registries;
-    this.state.dwDockerConfig.isLoading = isLoading;
-    this.state.dwDockerConfig.error = error;
-    return this;
-  }
-
-  public withCheDockerConfig(
-    registries: RegistryEntry[],
-    isLoading = false,
-    error?: string,
-  ): FakeStoreBuilder {
-    this.state.cheDockerConfig.registries = registries;
-    this.state.cheDockerConfig.isLoading = isLoading;
-    this.state.cheDockerConfig.error = error;
-    return this;
-  }
-
-  public withUserPreferences(
-    preferences: che.UserPreferences,
-    isLoading = false,
-  ): FakeStoreBuilder {
-    this.state.userPreferences.preferences = preferences;
-    this.state.userPreferences.isLoading = isLoading;
+    this.state.dockerConfig.registries = registries;
```

Comment:
Please rename this method: `withDwDockerConfig` -> `withDockerConfig`

Reply:
I have fixed it.